nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml

```diff
@@ -12,8 +12,8 @@ hepdata:
   url: "https://www.hepdata.net/record/ins1750330"
   version: 1
 
-version: 2
-version_comment: "Add uncertainty variant without luminosity uncertainty"
+version: 3
+version_comment: "Implement double differential distributions"
```

Comment:
Add the comment about the bugs here, this is the kind of thing that makes sense to keep as a comment!

Reply:
https://github.com/NNPDF/nnpdf/pull/2218/commits/f48776ae1b2b674c55a23821dab28422d8efd6c4